hosts.go

```diff
@@ -1,9 +1,9 @@
 package datadog
 
 type HostActionResp struct {
```

Comment:
While I do understand that you wanted to make this change to keep things consistent, I would rather not do it. It is backwards incompatible and while it's unfortunate that the code was written this way, let's keep the existing structure as it is to not break any depending code.